apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/columns.tsx

```diff
@@ -84,6 +86,39 @@ export const columns = [
     },
     header: 'Subject',
   }),
+  columnHelper.accessor('subject.type', {
+    cell: info => {
+      const subjectType = info.getValue();
+
+      return (
+        <TextWithNAFallback className={`max-w-[12ch]`}>{titleCase(subjectType)}</TextWithNAFallback>
+      );
+    },
+    header: 'Subject Type',
+  }),
+  columnHelper.accessor('subject.correlationId', {
+    cell: info => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks -- ESLint doesn't like `cell` not being `Cell`.
+      const { ref, styles } = useEllipsesWithTitle<HTMLSpanElement>();
+      const subjectId = info.getValue();
+
+      return (
+        <div className={`w-[11.8ch]`}>
+          <TextWithNAFallback
+            className={buttonVariants({
+              variant: 'link',
+              className: '!block cursor-pointer !p-0 text-sm !text-blue-500',
+            })}
+            style={styles}
+            ref={ref}
+          >
+            {subjectId}
+          </TextWithNAFallback>
+        </div>
+      );
```

Comment:
Should we have the full id in a tooltip or in the `title` attribute?

Reply:
Already has it.
![image](https://github.com/ballerine-io/ballerine/assets/13798202/5018524f-5584-47b4-acca-0471063e596c)